packages/yum/install_test.sh

```diff
@@ -15,10 +15,21 @@ gpgkey = https://rpm.mariadb.org/RPM-GPG-KEY-MariaDB
 gpgcheck = 1
 REPO
 
-sudo dnf install -y "https://packages.groonga.org/almalinux/${os_version}/groonga-release-latest.noarch.rpm"
-sudo dnf module -y disable mariadb
-sudo dnf module -y disable mysql
-sudo dnf install -y --enablerepo=powertools mariadb-server
+DNF_MODULE="dnf module -y"
+case "${os_version}" in
+  9)
+    DNF_INSTALL="dnf install -y"
+    sudo ${DNF_INSTALL} "https://apache.jfrog.io/artifactory/arrow/almalinux/${os_version}/apache-arrow-release-latest.rpm"
+    ;;
+  *)
+    DNF_INSTALL="dnf install -y --enablerepo=powertools"
+    sudo ${DNF_MODULE} disable mysql
+    ;;
```

Comment:
新しい方をデフォルトにして。10とか新しいバージョンが出たときに新しい方の処理を使いたいから。

```suggestion
  8)
    DNF_INSTALL="dnf install -y --enablerepo=powertools"
    sudo ${DNF_MODULE} disable mysql
    ;;
  *)
    DNF_INSTALL="dnf install -y"
    sudo ${DNF_INSTALL} "https://apache.jfrog.io/artifactory/arrow/almalinux/${os_version}/apache-arrow-release-latest.rpm"
    ;;
```

Reply:
なるほど。
新しい方をデフォルトにしました。